index.bs

```diff
@@ -50,12 +50,34 @@ Interfaces:
 * {{Event}}
 * {{EventTarget}}
 * {{File}}
+* {{FormData}}
+
+    Issue: The {{FormData}} constructor optionally takes {{HTMLFormElement}} and {{HTMLElement}} as parameters.
```

Comment:
Rather than adding these notes inline with the list, can we move these notes/details to a separate section and just link to them from here like a footnote?

Reply:
Bikeshed doesn't allow GFM-like footnotes, but we could indeed move the note about the Fetch fork to the end of the "common API index" section.

The `FormData` note is about the API requiring DOM elements, and I think it should probably stay inline.